src/wp-includes/blocks.php

```diff
@@ -2407,6 +2408,25 @@ function do_blocks( $content ) {
 	$top_level_block_count = count( $blocks );
 	$output                = '';
 
+	/**
+	 * Parsed blocks consist of a list of top-level blocks. Those top-level
+	 * blocks may themselves contain nested inner blocks. However, every
+	 * top-level block is rendered independently, meaning there are no data
+	 * dependencies between them.
+	 *
+	 * Ideally, therefore, the parser would only need to parse one complete
+	 * top-level block at a time, render it, and move on. Unfortunately, this
+	 * is not possible with {@see parse_blocks} because it must parse the
```

Comment:
Typically the function parens are included in such docs:

```suggestion
	 * is not possible with {@see parse_blocks()} because it must parse the
```

Reply:
updated. I never know on these how to feel because it seems misleading to have empty parentheses when the function takes arguments.